diff -urN deskbar-applet-0.8.3.orig/deskbar/module_list.py deskbar-applet-0.8.3/deskbar/module_list.py
--- deskbar-applet-0.8.3.orig/deskbar/module_list.py	2005-11-02 10:04:40.475960168 +0100
+++ deskbar-applet-0.8.3/deskbar/module_list.py	2005-11-02 10:04:54.054895856 +0100
@@ -47,7 +47,7 @@
 		"module-stopped" : (gobject.SIGNAL_RUN_LAST, gobject.TYPE_NONE, [gobject.TYPE_PYOBJECT]),
 	}
 	
-	def __init__ (self, dirs, extension=".py"):
+	def __init__ (self, dirs, extension=".pyc"):
 		"""
 		dirs: A list of directories to search. Relative pathnames and paths
 			  containing ~ will be expanded. If dirs is None the 
