diff -urN deskbar-applet-0.8.3.orig/deskbar/handlers/files.py deskbar-applet-0.8.3/deskbar/handlers/files.py
--- deskbar-applet-0.8.3.orig/deskbar/handlers/files.py	2005-11-02 20:31:00.513926016 +0100
+++ deskbar-applet-0.8.3/deskbar/handlers/files.py	2005-11-02 20:31:52.706991464 +0100
@@ -20,7 +20,7 @@
 
 class FileMatch(deskbar.handler.Match):
 	def __init__(self, backend, prefix, absname):
-		pixbuf = deskbar.handler_utils.load_icon_from_file(absname)
+		pixbuf = deskbar.handler_utils.load_icon_for_file(absname)
 		name = join(prefix, basename(absname))
 		deskbar.handler.Match.__init__(self, backend, name, pixbuf)
 		
